### DISCUSSION TITLE
Typescript typings missing

### DISCUSSION THREAD

**Discussion by n1ghtmare** [nextauthjs/next-auth#2898] [Help]
I've installed `next-auth@next` and although the documentation says that the typings are included in the project, `tsserver` is complaining that `Could not find the declaration file for module 'next-auth'`.

Can someone help me out here? Am I supposed to install something else?

**Comment by balazsorban44** [comment]
Could you try installing `next-auth@beta` instead?

https://next-auth.js.org/getting-started/upgrade-v4

**Reply by n1ghtmare** [reply]
First of all, thank you for replying, I really appreciate it.

Also, sorry I should've mentioned that I've tried `next-auth@beta` first. The issue was the same. Here is my `tsconfig.json`:

```
{
  "compilerOptions": {
    "incremental": true,
    "target": "es2016",
    "lib": [
      "dom",
      "dom.iterable",
      "esnext"
    ],
    "allowJs": true,
    "skipLibCheck": true,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "commonjs",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "baseUrl": "./",
    "strict": false
  },
  "include": [
    "next-env.d.ts",
    "**/*.ts",
    "**/*.tsx"
  ],
  "exclude": [
    "node_modules"
  ]
}

```

I've tried changing the `target` to be `es5`, as well as the `moduleResolution` to be `esnext`, the issue remains. Interestingly when I clone [next-auth-typescript-example](https://github.com/nextauthjs/next-auth-typescript-example) and try it in my editor I get the typings, as expected.

Not sure what I'm doing wrong, I read the documentation, but I can't spot my mistake.

**Reply by n1ghtmare** [reply]
Ok, I think I'm getting close to whatever the issue is. When I just install `next-auth@beta` and try it in my project, the typings work as expected. However, when I do `npm install @next-auth/prisma-adapter@next` the typings stop working. I'm really sorry if this was mentioned in the documentation and I've somehow missed that.

**Reply by n1ghtmare** [reply]
Actually, I did a `rm -rf node_modules`, reset my `package.json` and `package-lock.json` and tried both of the commands again `npm install --save next-auth@beta`, then `npm install @next-auth/prisma-adapter@next` and I get dependency errors (`unable to resolve dependency tree`).

I searched a bit more around the repo and found issues reported on this. Anyway, a way to fix it (for anyone looking) is to do `npm install @next-auth/prisma-adapter@next --force`.

Everything seems to be working so far.
